rllib/algorithms/algorithm.py

```diff
@@ -1312,7 +1344,9 @@ def training_step(self) -> ResultDict:
             # cases should use the multi-GPU optimizer, even if only using 1 GPU).
             # TODO: (sven) rename MultiGPUOptimizer into something more
             #  meaningful.
-            if self.config.get("simple_optimizer") is True:
+            if self.config.get("_enable_trainer_runner", False):
```

Comment:
same. Use AlgorithmConfig object syntax, not dicts.

Reply:
ok ok